jobs/bump-lockfile.Jenkinsfile

```diff
@@ -128,22 +127,25 @@ lock(resource: "bump-${params.STREAM}") {
             def parallelruns = [:]
             for (architecture in archinfo.keySet()) {
                 def arch = architecture
-                parallelruns[arch] = {
-                    if (arch == "x86_64") {
-                        shwrap("""
-                        cosa buildfetch --arch=${arch} --find-build-for-arch \
-                            --url=${BUILDS_BASE_HTTP_URL}/${branch}/builds
-                        cosa fetch --update-lockfile --dry-run
-                        """)
-                    } else {
-                        pipeutils.withExistingCosaRemoteSession(
-                            arch: arch, session: archinfo[arch]['session']) {
+                withCredentials([file(variable: 'AWS_CONFIG_FILE',
+                              credentialsId: 'aws-build-upload-config')]) {
```

Comment:
for reasons that aren't obvious I think this needs to go inside the `parallelruns[arch] = {` scope below. 

OR...

even better - let's move it to directly under the `stage("Fetch Metadata") {` scope above. 

basically it needs to not go out of scope before the function gets called (the `parallel parallelruns`)